tests/config/.gitignore

```diff
@@ -0,0 +1 @@
+/config.php
```

Comment:
I would move this entry to the `.gitignore` file in the project's root directory.